pkgs/by-name/vc/vcpkg/package.nix

```diff
@@ -32,6 +32,10 @@ stdenvNoCC.mkDerivation (finalAttrs: {
   installPhase = ''
       runHook preInstall
 
+      # If we don’t turn this off, then you won’t be able to run binaries that
+      # are installed by vcpkg.
+      find -name '*linux*.cmake' -exec bash -c 'echo "set(VCPKG_FIXUP_ELF_RPATH OFF)" >> "$1"' -- {} \;
```

Comment:
Can you be more specific than `'*linux*.cmake'`? In what directories are these files supposed to be?

Reply:
Oops. It was supposed to only search for files in the `triplets` directory. I just pushed a new version that fixes this issue.